README.md

```diff
@@ -8,13 +8,13 @@
 
 ## Introduction
 
-Add Mobile Center services to your app and collect crash reports and understand user behavior by analyzing the session, user and device information for your app. The SDK currently supports the following services:
+The Microsoft Mobile Center iOS SDK allows you to add Mobile Center services to your iOS application.
 
 1. **Analytics**: Mobile Center Analytics helps you understand user behavior and customer engagement to improve your iOS app. The SDK automatically captures session count, device properties like model, OS version etc. and pages. You can define your own custom events to measure things that matter to your business. All the information captured is available in the Mobile Center portal for you to analyze the data.
 
-2. **Crashes**: The Mobile Center SDK will automatically generate a crash log every time your app crashes. The log is first written to the device's storage and when the user starts the app again, the crash report will be forwarded to Mobile Center. Collecting crashes works for both beta and live apps, i.e. those submitted to App Store. Crash logs contain valuable information for you to help resolve the issue. Crashes uses PLCrashReporter 1.2.1.
+2. **Crashes**: Mobile Center Crashes will automatically generate a crash log every time your app crashes. The log is first written to the device's storage and when the user starts the app again, the crash report will be forwarded to Mobile Center. Collecting crashes works for both beta and live apps, i.e. those submitted to Google Play or other app stores. Crash logs contain viable information for you to help resolve the cause of the crash. The SDK gives you a lot of flexibility how to handle a crash log. As a developer you can collect and add additional information to the report if you like. Crashes uses PLCrashReporter 1.2.1.
```

Comment:
I want to have the part about the "restart" as early as possible because people will skip over it. I'll change the wording to make it closer to what you suggested, though.